src/services/api.ts

```diff
@@ -40,18 +41,22 @@ export class Api {
     }
 
     async post(apiUrl: string, data: any): Promise<any> {
-        const response = await fetch(`${this.baseUrl}${apiUrl}`, {
-            method: 'POST',
-            headers: {
-                'Content-Type': 'application/json',
-                Authorization: `Bearer ${this.jwtToken}`,
-            },
-            body: data ? JSON.stringify(data) : null,
-        })
+        const headers = {
+            'Content-Type': 'application/json',
+            Authorization: `Bearer ${this.jwtToken}`,
+        }
 
-        if (response.status !== 200 && response.status !== 204) {
-            throw new Error(response.statusText)
+        try {
+            const { data: responseData } = await axios({
```

Comment:
quick note: Axios can be configured with a based url and other fields (I didn't check exatly which one) to prevent using baseUrl each time

Reply:
For the moment `axios` is called only within `Api.post` method. Postponing enhancements in #413